.github/workflows/main.yml

```diff
@@ -15,17 +15,46 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       
-      - name: Build and push
+      - name: Build for linux/amd64
         uses: docker/build-push-action@v2
         with:
           build-args: |
             VCS_REF=${{ github.sha }}
-          platforms: linux/amd64,linux/arm64
-          tags: cs50/cli:${{ github.sha }},cs50/cli:latest
+          load: true
+          platforms: linux/amd64
+          tags: cs50/cli:${{ github.sha }}-amd64,cs50/cli:latest
```

Comment:
So we actually now have hundreds of tags that we don't really need. (Was more compelling for cs50/ide, where we might want to fixate.) Shall we stop creating these `cs50/cli:${{ github.sha }}-amd64` tags, and just do `:latest`?

Reply:
will do `cs50/cli:amd64` and `cs50/cli:latest` here because we're prioritizing pushing the amd64 image. `cs50/cli:amd64` is needed later in the workflow that we eventually will combine `cs50/cli:amd64` and `cs50/cli:arm64` to `cs50/cli:latest` as a multi-arch docker image.